docs/source/users/index.md

```diff
@@ -601,3 +601,26 @@ You can see a list of all aliases by running the `%ai list` command.
 Aliases' names can contain ASCII letters (uppercase and lowercase), numbers, hyphens, underscores, and periods. They may not contain colons. They may also not override built-in commands — run `%ai help` for a list of these commands.
 
 Aliases must refer to models or `LLMChain` objects; they cannot refer to other aliases.
+
+### Using the chat interface with SageMaker endpoints
+
+You can use the chat interface with models hosted using Amazon SageMaker.
+
+First, make sure that you've set your `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` environment variables either before starting JupyterLab or using the `%env` magic command within JupyterLab.
```

Comment:
`AWS_SESSION_TOKEN` might be internal only. Usually, `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are sufficient for most users. Rather than going into specific details which might differ between users, we should link to the AWS CLI documentation page, where all these options are listed and add a note that the access to AWS should be configured before proceeding with using the magics statement.
https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html